packages/edit-site/src/components/global-styles/custom-css.js

```diff
@@ -112,6 +112,8 @@ function CustomCSSControl( { blockName } ) {
 					onBlur={ handleOnBlur }
 					className="edit-site-global-styles__custom-css-input"
 					spellCheck={ false }
+					/// If we are customizing at the Block level we can include placeholder text with suggested syntax
```

Comment:
nit - extra / here and missing .
```suggestion
					// If we are customizing at the Block level we can include placeholder text with suggested syntax.
```